maps/adhomai/code/structures/bonfire.dm

```diff
@@ -157,4 +199,125 @@
 				add_overlay("fireplace_fire3")
 			if(1700 to 2000)
 				add_overlay("fireplace_fire4")
-		add_overlay("fireplace_glow")
\ No newline at end of file
+		add_overlay("fireplace_glow")
+
+/////////////
+// COOKING //
+/////////////
+
+/obj/item/weapon/reagent_containers/cooking_container/fire/
+	var/appliancetype
+
+/obj/item/weapon/reagent_containers/cooking_container/fire/pot
+	name = "pot"
+	shortname = "pot"
+	desc = "Chuck ingredients in this to cook something over a fire."
+	icon = 'icons/obj/food.dmi'
+	icon_state = "stew_empty"
+	appliancetype = POT
+
+/obj/item/weapon/reagent_containers/cooking_container/fire/pot/New(var/newloc, var/mat_key)
+	..(newloc)
+	var/material/material = get_material_by_name(mat_key ? mat_key : "iron")
```

Comment:
Why this is using material?

Reply:
I figured letting you make pots out of whatever material you have on hand would be useful. Only affects the name and maybe at some point the color.